src/Send/MemoField.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {TextInput} from '../components'
+import {COLORS} from '../theme'
+import {useSend} from './Context/SendContext'
+
+export const maxMemoLength = 256
+
+export const MemoField = () => {
+  const strings = useStrings()
+  const {memo, memoChanged} = useSend()
+
+  const showError = memo.length > maxMemoLength
+
+  return (
+    <View style={styles.container}>
+      <TextInput
+        value={memo}
+        onChangeText={(memo) => memoChanged(memo.trim())}
+        label={strings.label}
+        autoComplete={false}
+        testID="memoFieldInput"
+        errorText={showError ? 'error' : undefined}
+        noErrors={true}
+      />
+
+      <View style={styles.helper}>
+        <Text style={[styles.warning, showError && styles.error]}>{strings.warning}</Text>
+
+        <Text style={[styles.counter, showError && styles.error]}>{`${memo.length}/${maxMemoLength}`}</Text>
+      </View>
```

Comment:
https://github.com/Emurgo/yoroi-mobile/pull/2324#discussion_r1098017434

Reply:
Respecting the styles of the helpers

![image](https://user-images.githubusercontent.com/105349292/217335707-689d38f0-402a-4e6d-92cc-47695fb3f2f1.png)